test/checks/color/color-contrast.js

```diff
@@ -395,13 +395,25 @@ describe('color-contrast', function() {
         '\nx' +
         '\n</span></pre>'
     );
-    assert.doesNotThrow(function() {
+    assert.doesNotThrow(function () {
       contrastEvaluate.apply(checkContext, params);
     });
   });
 
-  describe('with pseudo elements', function() {
-    it('should return undefined if :before pseudo element has a background color', function() {
+  it('should return true for element which exactly overlaps midpoint of another element', function () {
```

Comment:
I feel this test is too indirect. I think I'd rather see us test getRectStack directly.